Taskfile.yml

```diff
@@ -558,7 +558,9 @@ tasks:
   # see https://github.com/DavidAnson/markdownlint-cli2#command-line for the reason we use double-quotes
   docs-fmt:
     desc: "Format and lint documentation"
+    deps: [init-tools]
```

Comment:
We expect people to run `task init` anyway, and that dependency slows things down. Let's remove it